src/screens/videos/mobile.js

```diff
@@ -22,30 +28,32 @@ export function VideosMobile({ className, playingVideoId, onReady, onPlay }) {
   return (
     <section
       id="videos"
-      className={cx("videos screen-content box-border", className)}
+      className={cx("videos screen-content box-border py-24", className)}
     >
-      <div className="full-screen h-full w-full flex-col">
-        <div className="split-screen h-screen-20vh flex w-full flex-col items-center justify-center text-center">
-          <h1>Videos you might like</h1>
-        </div>
+      <div className="relative mx-auto flex w-full max-w-7xl grow flex-col px-10">
+        <h2 className="text-center font-brand text-4xl text-white">
+          Videos you might like
+        </h2>
 
-        <div className="h-2/3">
-          <div className="flex h-full flex-row overflow-x-auto">
+        <div className="grow">
+          <div className="scrollbar-hide flex flex-row items-stretch overflow-x-auto">
             {videos.map((video) => (
               <div
                 key={video.id}
-                className="h-full shrink-0 basis-full p-5"
+                className="flex shrink-0 basis-full flex-col p-5"
                 ref={(el) => (videosRef.current[video.id] = el)}
               >
-                <div className="h-2/3">
-                  <YoutubeVideo
-                    videoId={video.id}
-                    className={cx(
-                      "h-full rounded-t-3xl",
-                      video.id !== playingVideoId && "hidden"
-                    )}
-                    onReady={onReady}
-                  />
+                <div className="grow">
+                  {playingVideoId === video.id ? (
+                    <YoutubeVideo
+                      videoId={video.id}
+                      className={cx(
+                        "h-full rounded-t-3xl",
+                        video.id !== playingVideoId && "hidden"
```

Comment:
`video.id !== playingVideoId` is always `false` here, because on line 47 we're checking for this condition to be true. A playing video probably shouldn't ever be hidden anyway 🙃

Reply:
Woops, that's a remnant from an older version of this component. Good catch!